libs/blog/become-author/feature-become-author-page/src/lib/components/become-author-advertisement/become-author-advertisement.component.html

```diff
@@ -0,0 +1,26 @@
+<al-card alGradientCard>
+  <div alCardContent class="@container">
+    <div class="@xl:flex-row flex flex-col">
+      <div>
+        <h2 class="border-al-gray-200 text-2xl font-bold lg:text-4xl">
+          How to become an angular.love Blogger?
+        </h2>
+        <p class="py-4 text-base">
+          Do you have an idea for an article? Would you like to start
+          contributing and creating the Angular Community with us?
+        </p>
+      </div>
+      <div
+        class="@xl:justify-end my-2 flex w-full flex-row items-center justify-center"
+      >
+        <a
+          class="bg-al-pink flex h-[46px] w-full items-center justify-center rounded-lg md:w-4/5 lg:w-full"
+          href="https://form.typeform.com/to/Zzf1Girt?typeform-source=angular.love"
```

Comment:
i am not sure about this href magic string, don't we want to have this configurable?

Reply:
No, AFAIK we only support english version. If this statement is true, we do not need any variable